core/mix/appup.ex

```diff
@@ -53,21 +53,15 @@ defmodule AntikytheraCore.Release.Appup do
     {only_v1, only_v2, diff_pairs} = :beam_lib.cmp_dirs(v1_ebin_dir, v2_ebin_dir)
     diff = reject_unchanged_modules(diff_pairs) |> Enum.map(fn {_f1, f2} -> f2 end)
 
-    # Croma automatically defines a module which might be common across multiple gears,
-    # e.g. `Elixir.Croma.TypeGen.Nilable.Antikythera.Url`.
-    # Such module must not be deleted when upgrading one of the gears
-    # because the other gears are still using that module.
-    only_v1_without_auto_generated =
-      Enum.reject(only_v1, fn beam_file ->
-        beam_file
-        |> read_module_name()
-        |> AntikytheraCore.Version.Gear.auto_generated_module?()
-      end)
-
     file_content = {
       v2_charlist,
       [
-        {v1_charlist, generate_instructions(only_v2, diff, only_v1_without_auto_generated)}
+        # Since some modules might be common across multiple gears,
```

Comment:
I think `some modules` should be `some generated modules` or `some modules which are generated by macro` or something is better. We assume that an Antikythera instance has common modules, i.g. libraries, in its `mix.exs`, so there is no problem in most cases. The exception is modules which are generated by macro.

Reply:
Thank you.
It's important to make our concerns as clear as possible.
I totally revised the comment.
https://github.com/access-company/antikythera/pull/200/commits/b7059e1d9b21b07341c3d4c6fc70311bb958c3d2